lib/plugins/physics.js

```diff
@@ -35,7 +35,12 @@ function inject (bot) {
   let lastSentYaw = null
   let doPhysicsTimer = null
   let lastPhysicsFrameTime = null
-  bot.physicEnabled = false
+  let shouldUsePhysics = false
+  bot.physicsEnabled = physicsEnabled ?? true
+  Object.defineProperty(bot, 'physicEnabled', {
```

Comment:
you can just remove that, this was added very recently no need to keep compat

Reply:
removed in 7d3c223